coba/tests/test_random.py

```diff
@@ -193,7 +193,7 @@ def std_shuffle() -> Iterator[Sequence[int]]:
             print(f"{coba_failure_rate}")
             print(f"{std_failure_rate}")
 
-        self.assertLess((coba_failure_rate-std_failure_rate)/std_failure_rate , .1)
+        self.assertLess((coba_failure_rate-std_failure_rate)/std_failure_rate , .2)
```

Comment:
@mrucker let me know if you have concerns with this bumps, maybe there's a way to signal unittest to retry 5-10 times until it passes, since its intermittent failure or another option might be to figure out how to filter out those tests from CI and just skip them.

Reply:
Alright done with the day and starting to look at this. Yeah, I'm fine with all your modifications. I always kind of assumed these tests might need to be skipped eventually. The code they test is changed very very rarely so I don't think it is worth dealing with the headache of unreliable tests. Hypothesis testing really wasn't designed for a constantly run test. I'm going to double check everything but it looks like everything is passing now? Is this good to merge as far as you know or is there something more that needs to be done?

Reply:
It's good to go, the perf tests are interesting as a smoke signal for regressions even if we increased the threshold in this PR. For now, this is better than nothing and later if they become annoying we can revisit. We're blocked on python 3.10 until sklearn releases a wheel for that version. FYI, commits straight to master also trigger this job - next to the commit you will see a checkmark or a cross depending if it passed or failed.

Reply:
It's also easy to re-trigger jobs if they fail and they take less than 2 minutes to run again so its not that bad.

Reply:
There's also only one skipped test in the CI, which looks like its tracking some bug to be fixed later so that's good.